### DISCUSSION TITLE
Is it possible to connect two sandboxes?

### DISCUSSION THREAD

**Discussion by m0rph3usX** [sandboxie-plus/Sandboxie#3838] [Q&A Feedback] [answered]
Hey there,

my idea is as follows:

1. create sandbox "A" and install my favourite IDE / SDK
2. create a sandbox "B" with third party libraries that can be used by sandbox "A"
3. create a sandbox "C" with third party libraries (same as "B" but with another version) that can be used by sandbox "A" 

So now it would be cool if I could switch between Sandbox
"A" accessing "B" 
or
"A" accessing "C". 

So I could try different versions of the third party libraries without reinstalling them all the time.

Is there such a feature I missed?
Thanks in advance!

**Comment by offhub** [comment, marked as answer]
You can try the [Snapshots](https://sandboxie-plus.github.io/sandboxie-docs/PlusContent/BoxSnapshots.html) feature.

1. Create a box
2. Install the IDE / SDK
3. Take a snapshot (**snapA**, _baseline_)
4. Install 3rd party libs
5. Take a snapshot (**snapB**)
6. Go to **SnapA**
7. Install 3rd party libs
8. Take a snapshot (**snapC**)
9. You can switch between **snapB** and **snapC** with the "go to snapshot" button.

**Reply by m0rph3usX** [reply]
Thanks, for the tip, this should work.

The only disadvantage of using snapshot:
If I want to update sandbox "A" I have to do it for "B" and "C" instead "A" only.
